engine/src/main/resources/assets/shaders/vignette_vert.glsl

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright 2012 Benjamin Glatzel <benjamin.glatzel@me.com>
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+void main()
+{
+	gl_Position = ftransform();
```

Comment:
Tabs and spaces together in a single file? Uh oh.

We usually don't use tabs anywhere, as I'm sure you know by now :)